lib/trino-record-decoder/pom.xml

```diff
@@ -16,6 +16,16 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>confluent</id>
+            <url>https://packages.confluent.io/maven/</url>
```

Comment:
Will this make us hit confluent maven repo (first, before maven central) for all artifacts needed by this module?

Reply:
Took a look at the effective pom for this module. It is listed first so I think that means Maven will hit is first to find artifacts.

Reply:
yes, and that's not desireable. 

pre-existing (trino-pinot, trino-kafka), so no change requested